array-manipulation-2/src/drop-right.js

```diff
@@ -0,0 +1,23 @@
+/* exported dropRight */
+
+/*  function declaration with parameter: array (an array)
+      and count(number of items in array to disgard)
+    variable declaration for final array to return (is empty at first)
+    for loop where i is indexes of parameter array and condition statement is
+      that i is less than the array.length minus the count
+    also need to take into account for empty array parameter: just return an empty array
+    otherwise push the array[i] into the final array
+    return the final array from the function
+*/
+
+function dropRight(array, count) {
+  var finalArr = [];
+  for (var i = 0; i < array.length - count; i++) {
+    if (array.length === 0) {
+      finalArr = [];
```

Comment:
This conditional isn't necessary. If the array has a length of zero, then the loop would never start, which would mean you return the initial value of `finalArr`, which is an empty array.